src/vs/workbench/contrib/debug/browser/replFilterView.ts

```diff
@@ -0,0 +1,180 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
As far as I understand nothing in this calss is REPL specific.
Is this a 1:1 copy from the Problems? If yes we should just have a shared class
fyi @sandy081